.github/workflows/code-quality.yml

```diff
@@ -32,14 +32,19 @@ jobs:
       - name: Install dependencies
         run: uv sync --dev
 
-      - name: Run ruff linting
-        run: uv run ruff check .
-
-      - name: Run ruff formatting check
-        run: uv run ruff format --check .
+      - name: Set PY
```

Comment:
What does `Set PY` mean? Setting the python version?

Reply:
Yep, @ryanprior is correct